bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/AbstractConfiguration.java

```diff
@@ -70,6 +70,11 @@
     // Client auth provider factory class name. It must be configured on Bookies to for the Auditor
     protected final static String CLIENT_AUTH_PROVIDER_FACTORY_CLASS = "clientAuthProviderFactoryClass";
 
+    // Common SSL configuration
+    protected final static String SSL_ENABLED_CIPHER_SUITES = "sslEnabledCipherSuites";
+    protected final static String SSL_ENABLED_PROTOCOLS = "sslEnabledProtocols";
```

Comment:
the option will be reflected to SSLEngine#setEnableProtocols.
I will update the comments on setters and here